world/stat_checks/check_commands.py

```diff
@@ -50,6 +54,22 @@ def do_normal_check(self):
             self.caller, stat=stat, skill=skill, rating=rating
         )
 
+    def do_private_check(self):
+        receiver_names = [player for player in self.rhs.split(",")]
+        receiver_list = []
```

Comment:
just as an fyi there's already an attribute in classes that does this: `self.rhslist` is the arguments on the right hand side of the `=` that are comma-separated